packages/scss/src/components/status/component.scss

```diff
@@ -0,0 +1,47 @@
+@mixin component {
+	display: inline-flex;
+	align-items: center;
+	white-space: nowrap;
+
+	@at-root {
+		.status-dot {
+			aspect-ratio: 1;
+			width: var(--spacings-smaller);
+			border-radius: 50%;
+			background-color: currentColor;
+			margin: calc(var(--spacings-smallest) / 2);
+			margin-right: var(--spacings-smaller);
+			position: relative;
+			z-index: 1;
+			display: inline-flex;
+		}
+
+		.status-dot-important {
+			@keyframes status {
```

Comment:
with @RipripKipkip we did this animation : 
@keyframes pulse {
  0% {
    box-shadow: 0 0 0 0 rgba(var(--colors-error-600-rgb), 1);
  }
  70% {
    box-shadow: 0 0 0 6px rgba(var(--colors-error-200-rgb), 0);
  }
  100% {
    box-shadow: 0 0 0 0 rgba(var(--colors-error-200-rgb), 0);
  }
}